packages/textlint-fixer-formatter/src/index.ts

```diff
@@ -46,7 +45,11 @@ ${ex}`);
     };
 }
 
-export function getFormatterList() {
+export interface FixerFormatterDetail {
+    name: string;
+}
+
+export function getFormatterList(): FixerFormatterDetail[] {
```

Comment:
For consistency, `getFormatterList()` could be renamed to `getFixerFormatterDetail()`. Could you let me know your thoughts.

package | API | returns
-|-|-
`textlint-formatter` | `getFormatterList()` | `FormatterDetail[]`
`textlint-fixer-formatter` | `getFormatterList()` or `getFixerFormatterList()`? | `FixerFormatterDetail[]`

Reply:
> getFixerFormatterList

Reasonable.
I've fixed it.